x-pack/plugins/maps/public/actions/store_actions.js

```diff
@@ -61,6 +63,8 @@ export const DRAW_TYPE = {
 export const SET_SCROLL_ZOOM = 'SET_SCROLL_ZOOM';
 export const SET_MAP_INIT_ERROR = 'SET_MAP_INIT_ERROR';
 
+export const DEFAULT_COLORS = ['#e6194b', '#3cb44b', '#ffe119', '#f58231', '#911eb4'];
```

Comment:
We should probably store these either in the `common/constants.js` file or something similar under `common/`